das/coordinator_test.go

```diff
@@ -231,13 +266,18 @@ func TestCoordinator(t *testing.T) {
 }
 
 func BenchmarkCoordinator(b *testing.B) {
-	concurrency := 100
-	samplingRange := uint64(10)
 	timeoutDelay := 5 * time.Second
 
+	params := dasingParams{}
+	params.samplingRange = 10
+	params.concurrencyLimit = 100
+	params.bgStoreInterval = 10 * time.Minute
+	params.priorityQueueSize = 16 * 4
+	params.genesisHeight = 1
```

Comment:
You can just take the default and only change the fields you want different from the default params